todo_app/app.py

```diff
@@ -2,7 +2,9 @@
 from flask import render_template
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items, add_item
+from todo_app.data.session_items import add_item
+from todo_app.data.trello_items import get_items
```

Comment:
When you are done with this exercise, you should be able to delete any mention of session_items and delete the session_items.py file itself.

Currently your app doesn't make sense - you can try to create new items, but they get added to the "session" and never get displayed.

All functionality should be moved over to Trello